lib/src/operance_data_column_header.dart

```diff
@@ -255,65 +269,67 @@ class _ColumnHeader<T> extends StatelessWidget {
   /// Indicates whether the column is being dragged.
   final bool dragging;
```

Comment:
should we rename this to `active`? I called it dragged when i was initially creating the draggable behavior but forgot to rename it when i decided not everyone wants draggable columns. The boolean represents the active color state and nothing else really. dragging seems to imply more than just a color state change

Reply:
Yeah, I believe we should. It would make it a lot more semantically correct and easier to understand for devs. Can't particularly see any downsides in the rename